test/Dockerfile

```diff
@@ -33,7 +31,9 @@ RUN uname -a
 
 RUN export | sed 's/\(NPM_TOKEN=....\).*\(....\)/\1___\2/g'
 
-RUN npm install
+RUN --mount=type=ssh \
+    --mount=type=secret,id=NPM_TOKEN,uid=1000 \
+    secrets2env npm install
```

Comment:
Is this something that needs to be copied into all Dockerfiles?

Reply:
Yes, that would be nice.